examples/resources/rafay_break_glass_access/providers.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_providers {
+    rafay = {
+      version = ">= 0.1"
+      source  = "registry.terraform.io/Rafay/rafay"
```

Comment:
it should be `registry.terraform.io/RafaySystems/rafay`

Reply:
updated it